framework/harness/harness.go

```diff
@@ -185,6 +185,14 @@ func startServer(port int, handler http.Handler) error {
 	server := &http.Server{
 		Addr: fmt.Sprintf(":%d", port),
 		Handler: http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			w.Header().Set("Access-Control-Allow-Origin", "*")
+			w.Header().Set("Access-Control-Allow-Methods", "GET, POST, PUT, DELETE, OPTIONS")
+			w.Header().Set("Access-Control-Allow-Headers", "*")
+
+			if r.Method == http.MethodOptions {
```

Comment:
Handle options requests instead of just HEAD requests.

Reply:
Could help to have a comment saying something like "// Status isn't required on OPTIONS request" (at least, I did a double take because I wasn't familiar with options)